easybuild/tools/module_generator.py

```diff
@@ -52,24 +55,28 @@ class ModuleGenerator(object):
     """
     Class for generating module files.
     """
+    SYNTAX = None
 
     # chars we want to escape in the generated modulefiles
     CHARS_TO_ESCAPE = ["$"]
+    MODULE_SUFFIX = ''
```

Comment:
should be renamed to `MODULE_FILE_EXTENSION`, Tcl module files have no extension, Lua module file have `.lua`